finalfrontier-utils/build.rs

```diff
@@ -0,0 +1,32 @@
+use git2::{DescribeFormatOptions, DescribeOptions, Repository};
+
+fn main() {
+    if let Ok(repo) = Repository::open_from_env() {
+        let describe = repo
+            .describe(
+                DescribeOptions::new()
+                    .describe_tags()
+                    .show_commit_oid_as_fallback(true),
+            )
+            .expect("Could not get description for git repo.");
+
+        let desc = describe
+            .format(Some(
+                DescribeFormatOptions::new()
+                    .always_use_long_format(true)
+                    .dirty_suffix("-dirty"),
+            ))
+            .expect("Could not format description for git repo.");
+        println!("cargo:rustc-env=FINALFRONTIER_GIT_DESC={}", desc);
```

Comment:
Is this used in `-utils`?

Reply:
Don't think so, that's probably copy-pasted from the library's `build.rs`.